feat: resize all images to the same aspect ratio and format (#701)
ansys/pyansys: doc/source/_static/thumbnails/README.md

# Thumbnails

This directory contains thumbnails for the documentation. They are used in the
documentation to provide a visual representation of the various PyAnsys libraries
and tools.

## Resizing

> All thumbnails should be 640x480 pixels. This is the size that the documentation
> expects and is resized to this size if it is not already. Resize takes place

Reviewer: ```suggestion
All thumbnails should be 640x480 pixels. This is the size required by the
documentation. If they are not already this size, thumbnails will be resized to
640x480 pixels. Resize takes place
```
Minor suggestion to ease reading

Author: Thanks @SMoraisAnsys!
